nexus/subscription.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: nexus-operator-m88i
+  namespace: openshift-operators
+spec:
+  channel: alpha
+  name: nexus-operator-m88i
+  source: community-operators
+  sourceNamespace: openshift-marketplace
```

Comment:
Switched from using the direct operator yaml in prior commit ( https://github.com/rnc/jbs-demo/pull/1/commits/7d5f4e060ee4fcbfa24aa03ee8ad3da422229554#diff-123e2f1525b13cf31e179bf2d282ad89c101871e1a20900438337fd5323f9729 ) which was from https://github.com/m88i/nexus-operator/blob/main/nexus-operator.yaml to a subscription.